### PR TITLE
Digisam 140 header and search in patternlab

### PR DESCRIPTION
**Pull request by jara2**

### CODE REVIEW

File: public/assets/fonts/Noway-Bold-Italic-webfont/generator_config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Font Squirrel Font-face Generator Configuration File
```
[the remark this answers is not in the document]
Comment (jara2) [reply public/assets/fonts/Noway-Bold-Italic-webfont/generator_config.txt]: I deleted a big .zip file not needed. Not sure about configs but they don't take much space.

File: src/components/Header.vue [hunk 2]
```diff
@@ -1,84 +1,128 @@
...
 
 export default {
   name: 'Header',
   components: {
-    Search
+    Search,
+    SearchMobile
+  },
+  mounted () {
+    let scripts = document.createElement('script')
```
[the remark this answers is not in the document]
Comment (jara2) [reply src/components/Header.vue]: Agree. But I made a separate task for this.. DIGISAM-154

File: package.json [hunk 3]
```diff
@@ -9,7 +9,9 @@
   },
   "dependencies": {
     "axios": "^0.19.0",
-    "bootstrap-vue": "^2.0.2",
+    "foundation-sites": "^6.4.4-rc1",
```
[the remark this answers is not in the document]
Comment (jara2) [reply package.json]: yes

File: public/assets/css/style-discovery.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.bar.alert-bar {
```
[the remark this answers is not in the document]
Comment (jara2) [reply public/assets/css/style-discovery.css]: Yes, removed it

File: src/components/Header.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (jorntx) [reply src/components/Header.vue]: Good stuff

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (jorntx) [reply package.json]: check

**Pull request merged**